app/realtime/card_realtime.rb

```diff
@@ -7,7 +7,7 @@ def self.card_moved(card, move_params)
     )
   end
 
-  def self.card_created(card, html)
+  def self.card_created(card, originated_by, html)
```

Comment:
Unused method argument - `originated_by`. If it's necessary, use `_` or `_originated_by` as an argument name to indicate that it won't be used.